roles/sriov_config/tasks/check_sriov_network.yml

```
@@ -0,0 +1,12 @@
+---
+- name: Wait until NetworkAttachmentDefinition is created from SriovNetwork {{ sriov_conf.network.name }}
+  community.kubernetes.k8s_info:
+    api_version: k8s.cni.cncf.io/v1
+    name: "{{ sriov_conf.network.name }}"
+    namespace: "{{ sriov_conf.network.network_namespace }}"
```

Since ns is not required, default to "default" ns
```suggestion
    namespace: "{{ sriov_conf.network.network_namespace | default('default') }}"
```

---

ack